api/can_api_v2_definition.py

```diff
@@ -303,6 +310,12 @@ class Annotations(base_model.APIBaseModel):
     vaccinationsCompletedRatio: Optional[FieldAnnotations] = pydantic.Field(
         None, description="Annotations for vaccinationsCompletedRatio"
     )
+    vaccinationsAdditionalDoseRatio: Optional[FieldAnnotations] = pydantic.Field(
+        None,
+        description=(
+            "Ratio of population that are fully vaccinated and received a booster (additional) dose."
```

Comment:
nits:
* "received" => "have received" (to match dose count description above)
* "additional" => "or additional" (to match dose count description above)

Though TBH I'm not sure if "additional" or "or additional" makes more sense. What does CDC do?

Reply:
looks like [they use](https://data.cdc.gov/Vaccinations/COVID-19-Vaccinations-in-the-United-States-County/8xkx-amqh) "or additional", I'll make this match